docs/development/documentation_guidelines.rst

```diff
@@ -60,10 +85,10 @@ a reliable internet connection:
       make docs-linkcheck
 
 The :ref:`CI tests<ci_tests>` by default create staging servers to test the
```

Comment:
You can also remove this sentence, as these tests no longer run in the `securedrop-docs` repo.